dashboard/pom.xml

```
@@ -26,6 +26,11 @@
 
     <artifactId>dashboard</artifactId>
     <name>Apache Uniffle Dashboard</name>
+    
+    <properties>
+        <nodeDownloadRoot>https://nodejs.org/dist/</nodeDownloadRoot>
+        <npmDownloadRoot>https://registry.npmjs.org/npm/-/</npmDownloadRoot>
```

What about:
```suggestion
        <nodeDownloadUrl>https://nodejs.org/dist/</nodeDownloadUrl>
        <npmDownloadUrl>https://registry.npmjs.org/npm/-/</npmDownloadUrl>
```

---

Why not the original name? It makes sense as it is?

---

Is download root a conventional concept ? This looks like a repository URL, which is used as the root to construct URLs to fetch packets from. I just think `Root` is a bit misleading and unnecessarily specific here. What about

```suggestion
        <nodeRepositoryUrl>https://nodejs.org/dist/</nodeRepositoryUrl>
        <npmRepositoryUrl>https://registry.npmjs.org/npm/-/</npmRepositoryUrl>
```

---

Either works for me